test/outputs.jl

```diff
@@ -83,6 +83,11 @@ include(joinpath(TESTDIR, "testutils.jl"))
     tmle = failed_estimate["TMLE"]
     @test tmle["PVALUE"] === NaN
     @test tmle["EFFECT_SIZE"] == "Failed"
+
+    # Check subsetted results & optional QQplot generation
```

Comment:
In the comment of this test, good to state exactly what problem this is checking. I assume sub_results only contains a NA because of the issue, but in 6 months time I will have forgotten :)